x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -0,0 +1,137 @@
+[role="xpack"]
+
+:type: httpjson
+
+[id="{beatname_lc}-input-{type}"]
+=== HTTP JSON input
+
+++++
+<titleabbrev>HTTP JSON</titleabbrev>
+++++
+
+beta[]
+
+Use the `httpjson` input to read messages from an HTTP API that provides JSON data.
+
+This input can, for example, be used to receive MISP threat indicators, as in the 
+filebeat x-pack misp module.
+
+This input supports repeated retrieval at a configurable interval, and it also 
+supports pagination.
```

Comment:
The input supports timed data retrieval at a configurable interval and pagination.